rest_service.go

```diff
@@ -68,6 +69,8 @@ func (s *Service) App() *gimlet.APIApp {
 	app.AddRoute("/download/mongodb").Version(1).Post().Handler(s.downloadMongoDB)
 	app.AddRoute("/list/{filter}").Version(1).Get().Handler(s.listProcesses)
 	app.AddRoute("/list/group/{name}").Version(1).Get().Handler(s.listGroupMembers)
+	app.AddRoute("/list/oom").Version(1).Delete().Handler(s.oomTrackerClear)
```

Comment:
Is there a reason this is also under /list? Strange route for a Delete.

Reply:
it's saying delete everything in the list? I'm not sure I can see an option that I like more though. 

I often feel like there are no right answers in REST APIs.